### ISSUE TITLE
Job-Preview (web) should not require getPlan() to work with empty args

### ISSUE THREAD

**Issue by andrehacker** [stratosphere/stratosphere#336]
The job preview in the job-submission web interface only works if the getPlan() method returns a plan for empty args. I think this is not-so-nice because:
- the user does not know this and wonders why the job-preview is not working (the error is not very helpful)
- some arguments are mandatory and it is sometimes hard to get a plan without them (had this problem in one of my jobs)
- it requires to write a lot of otherwise unnecessary code like `args.length > 0 ?...` (or is there another reason why we need this?)

This also caused issue #335.

However, I don't have any proposal how to do it better;-)

**Comment by rmetzger** [comment]
How about adding some text in the area of the Job-preview that explains everything to the user!

**Comment by andrehacker** [comment]
Good idea! First, we could add a default text like "Please select a job", as long as the user didn't select a job. And if he selected one, but it throws an exception, we can show another message (we can give him a hint to look at the examples).
Good if someone else can do this, I want to continue with accumulators.
